packages/webpack-stats-report/src/createReport.ts

```diff
@@ -150,20 +161,17 @@ export function createNoChangeReport(reportData: ReportData[]): string {
   )}\n</details>`;
 }
 
-function getReducedOrIncreased(
-  diffSize: number
-): string {
-  return diffSize > 0
-    ? "increased"
-    : "reduced";
+function getReducedOrIncreased(diffSize: number): string {
+  return diffSize > 0 ? "+" : "-";
 }
 
 function formatBytes(bytes: number, decimals: number = 2): string {
   const inKb = bytes / 1000;
```

Comment:
I know it's not you, but 1000 instead of 1024 makes me sad.

Reply:
Nice catch. Updated.